profiles/tests/test_activity.py

```diff
@@ -18,10 +18,18 @@ def test_get_all_activities_unauthorized(self):
         response = self.client.get("/api/activities/")
         self.assertEqual(200, response.status_code)
 
-    def test_get_all_activities_data(self):
-        response = self.client.get("/api/activities/")
-        self.assertEqual([{'id': 46, 'name': 'sale'}, {'id': 47, 'name': 'transport'}, {'id': 48,
-                            'name': 'education'}, {'id': 49, 'name': 'medicine'}], response.json())
+    def test_get_activity_data_authorized_admin(self):
+        self.client.force_authenticate(self.test_person_is_admin)
+        response = self.client.get(f"/api/activities/{self.sale_activity.id}/")
```

Comment:
I recommend to use `.format()` with named arguments instead of f-strings.

Reply:
... in this case.

Reason: properly named placeholders usually give clearer understanding what is in this part of URL, than some value directly put into f-string.

Reply:
Why?